Intro to Data Science videos.Rmd

```diff
@@ -0,0 +1,8 @@
+# Intro to Data Science -- videos
```

Comment:
- The .Rmd filename should be words only and joined with underscores, no white space. (Update: It does not need to be the same as the branch name.)
- The .Rmd filename can only contain lowercase letters. (Otherwise the filenames do not sort nicely on the repo home page.)